data/changelog.txt

```diff
@@ -360,6 +391,7 @@ Adds check enforcing almost-never-auto convention.
 Adds a GitHub workflow running flake8 to check the Python scripts.
 Consistently styles code in CMakeLists.txt files.
 Adds CMake linting.
+Captures ODR violations using GOLD linker, ASAN or LTO.
 
 ## I18N and A11Y:
 Fixes translations of martial art initiate and computer terminal messages.
```

Comment:
Shouldn't `Places cursor at highlighted item name in construction and crafting menus, as expected by screen readers.` be here, not under Bugfixes?

Reply:
Seems reasonable, I'll move it.